plumbing/transport/common.go

```diff
@@ -228,10 +229,10 @@ func parseSCPLike(endpoint string) (Endpoint, bool) {
 	}
 
 	m := scpLikeUrlRegExp.FindStringSubmatch(endpoint)
-	return &scpEndpoint{
-		user: m[1],
-		host: m[2],
-		path: m[3],
+	return &SCPEndpoint{
+		Username: m[1],
+		Hostname: m[2],
+		Pathname: m[3],
```

Comment:
I cannot, for the same reason as I needed to rename the other fields -- they conflict with the method names.